riot/ble/ble_scan/ble_scan.ipynb

```diff
@@ -6,14 +6,16 @@
    "source": [
     "## Scan BLE advertisement paquets\n",
     "\n",
-    "In this exercise, you will use one [nRF52DK](https://www.iot-lab.info/docs/boards/nordic-nrf52dk/) to scan BLE advertisement paquets sent by 2 other nRF52DK boards.\n",
+    "In this exercise, you will use one [nrf52dk](https://www.iot-lab.info/docs/boards/nordic-nrf52dk/) to scan BLE advertisement paquets sent by 2 other nrf52dk boards.\n",
```

Comment:
the names are correct, these are the brand names.

Reply:
done!